tool/teleport/common/teleport.go

```diff
@@ -477,6 +483,14 @@ func onConfigDump(flags dumpFlags) error {
 		return trace.Wrap(err)
 	}
 
+	entries, err := os.ReadDir(flags.DataDir)
+	if !os.IsNotExist(err) && len(entries) != 0 {
+		fmt.Fprintf(
+			os.Stderr,
+			"The data directory %s has existing cluster state. Running this configuration is likely a mistake. To join a new cluster, specify an alternate --data-dir or clear the %s directory.\n",
```

Comment:
> has existing cluster state

Technically, all we know is the directory is not empty, maybe it has some other data in it. I would probably just say "The data directory %s is not empty".

Reply:
Changed it to `The data directory %s is not empty and may contain existing cluster state`